File: Paradigmas de programacion 2do Año/Trabajo Practico N3 Paradigmas de programacion/ejercicio1.py
# Nombre alumno: Franco Genaro Reyes
class Perro:
    def __init__(self, nombre, edad, raza):
        self.edad = edad
        self.raza = raza
        self.nombre = nombre  

lista_perros = []
edad_mayor = 0

for i in range(6):
    nombre = input(f'Ingresá el nombre del perro {i+1}: ')
    edad = int(input(f'Ingresá la edad de {nombre}: '))
    raza = str(input(f'Ingresá la raza de {nombre}: '))
    perro = Perro(nombre, edad, raza)
    lista_perros.append(perro)
    
edad_mayor = lista_perros[0].edad

for i in range(1, len(lista_perros)):
    if lista_perros[i].edad > edad_mayor:
        edad_mayor = lista_perros[i].edad

print(edad_mayor)

File: Paradigmas de programacion 2do Año/Trabajo Practico N7 Paradigmas de programacion/Ejercicio1.java
// Nombre alumno: Franco Genaro Reyes

import java.util.Scanner;

public class Ejercicio1 {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);  // Paso 1
        System.out.print("Ingrese una palabra: ");
        String palabra = input.nextLine();       // Leer palabra

        for (int i = 0; i < palabra.length(); i++) {  // Paso 2
            // Paso 3: imprimir espacios
            for (int j = 0; j < i; j++) {
                System.out.print(" ");
            }
            // Paso 4: imprimir letra
            System.out.println(palabra.charAt(i));
        }

        input.close();
    }
}
